### ISSUE TITLE
How to train with my own dataset?

### ISSUE THREAD

**Issue by Daisy-7** [justchenhao/STANet#23]
Thanks for your work.
I want to use your model for training, and at the same time change the learning rate drop method and data augmentation method during training. What should I do?

**Comment by xyl-507** [comment]
作者您好，您们的数据集网页我登不进去，能不能请你上传一下百度云的链接，谢谢。真的很想学习一下变化检测。

**Comment by justchenhao** [comment]
Thanks for your attention. You can modify the `optimizer` in the model file and the augmentation functions in `base_dataset.py`.

**Comment by justchenhao** [comment]
Close for now.
